base/fs.cpp

```diff
@@ -210,6 +210,47 @@ std::string get_file_title_with_path(const std::string& filename)
   return filename;
 }
 
+std::string get_relative_path(const std::string& filename, const std::string& base_path)
+{
+  std::vector<std::string> baseDirs; 
```

Comment:
There is a trailing white space here:

![image](https://github.com/aseprite/laf/assets/39654/6df48db7-aa96-4b5f-879d-ff22c9cdfefe)